File: airport_routes/v1/get/Node SDK/airport_routes.js
const WAConvo = require("waconvo");

const waconvo = new WAConvo({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // Find all destinations served by CDG Airport
    const response = await waconvo.airport.directDestinations.get({
      departureAirportCode: "MAD",
    });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();

File: airport_routes/v1/get/Python SDK/airport_routes.py
# Install the Python library from https://pypi.org/project/waconvo
from waconvo import Client, ResponseError

waconvo = Client(
    client_id='YOUR_WACONVO_API_KEY',
    client_secret='YOUR_WACONVO_API_SECRET'
)

try:
    '''
    What are the destinations served by MAD airport?
    '''
    response = waconvo.airport.direct_destinations.get(departureAirportCode='MAD')
    print(response.data)
except ResponseError as error:
    raise error
